KIPs/kip-34.md

```diff
@@ -390,7 +393,8 @@ Each keyring class uses the `PrivateKey` class, which has one private key as a m
 | Method | Description |
 | ----------- | ----------- |
 | getPublicKey(): List&#60;String&#62; | Returns the public key strings. |
```

Comment:
@jimni1222 Is this function remained?

Reply:
This is remained because `compressed` parameter is optional. 
For java, to support optional parameter, function should be overloaded.